docs/plugins/discovery-azure.asciidoc

```diff
@@ -1,36 +1,36 @@
-[[discovery-azure]]
+[[discovery-azure-classic]]
 === Azure Discovery Plugin
 
-The Azure Discovery plugin uses the Azure API for unicast discovery.
+The Azure Classic Discovery plugin uses the Azure Classic API for unicast discovery.
```

Comment:
The alternative is not built yet. That's why I can't link to it. :p 

For sure this will need to be updated when #19146 will be implemented.